From 00fc691802e671594cea1d0a5e880d255e04b7f0 Mon Sep 17 00:00:00 2001
From: Quirin Gylstorff <quirin.gylstorff@siemens.com>
Date: Wed, 3 May 2023 16:02:37 +0200
Subject: [PATCH] d/rules: Add option for suricatta_lua

This allows to use lua plugins as suricatta backends. It is optional
as it overrides the default HAWKBIT backend.

Signed-off-by: Quirin Gylstorff <quirin.gylstorff@siemens.com>
---
 debian/rules | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/debian/rules b/debian/rules
index 57da4ca7..91106fbd 100755
--- a/debian/rules
+++ b/debian/rules
@@ -56,6 +56,9 @@ endif
 ifneq (,$(filter pkg.swupdate.p11,$(DEB_BUILD_PROFILES)))
 	echo CONFIG_PKCS11=y >> configs/debian_defconfig
 endif
+ifneq (,$(filter pkg.swupdate.suricattalua,$(DEB_BUILD_PROFILES)))
+	echo CONFIG_SURICATTA_LUA=y >> configs/debian_defconfig
+endif
 ifneq (,$(LUA_VERSION))
 	echo CONFIG_LUAPKG=\"lua$(LUA_VERSION)\" >> configs/debian_defconfig
 	echo CONFIG_LUASCRIPTHANDLER=y           >> configs/debian_defconfig
-- 
2.39.2

